### PR TITLE
Document required snap connections in new README.md

### PR DESCRIPTION
**Pull request by lathiat**
Create a new README.md with basic snap documentation including the
snap connections required for the exporter to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Prometheus OVN Exporter Snap
+
+This repository contains the source for [prometheus-ovn-exporter](https://snapcraft.io/prometheus-openstack-exporter), an unofficial snap of the [OVN Exporter](https://github.com/greenpau/ovn_exporter) for
```
Comment (fnordahl) [inline README.md]: Just curious about what to read into the word `unofficial` here, unofficial as in not endorsed by upstream? From distribution point of view it is owned by the `Canonical` publisher, which kind of is "official".

Would there be a different word that is more fitting?

**Review by fnordahl — COMMENTED** [review]
Thank you so much for the contribution, the repository does indeed deserve a README!

I have one comment/question in-line that I'd like to hear your thoughts on before proceeding.

File: README.md (hunk 1 above)
Comment (lathiat) [reply README.md]: To be honest, I copied this wording from another snap repo. I guess it stems from the original push that snap packaging should be in upstream git repos. I'm happy to remove it.

Comment (lathiat) [reply README.md]: Pushed a new revision with updated wording

**Review by fnordahl — APPROVED** [review]
LGTM, thanks!

**Pull request merged**